internal/factsengine/gatherers/cibadmin.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 const (
-	CibAdminGathererName  = "cibadmin"
-	CibAdminGathererCache = "cibadmin"
+	CibAdminGathererName  = "/usr/sbin/cibadmin"
```

Comment:
Careful!
Don't touch this. This references the name of the gatherer and the name of the entry in the cache

Reply:
Fixed in https://github.com/trento-project/agent/pull/421/commits/14909a1fcde6eb56f24e95d4298db38f2537f1b5